doc_src/auto_examples/README.rst

```diff
@@ -0,0 +1,5 @@
+This is the Franck lab gallery
+==============================
+
+Below is a gallery of examples using our 
+various functions.
```

Comment:
@guinneak198 : did you create this file, or was it auto-generated?

Reply:
I created this. This is what is displayed at the top of the examples page